release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -47,6 +47,29 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-10-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-10-OCP-on-ARM"]
+==== {product-title} on ARM
+
+{product-title} 4.10 is now supported on ARM based AWS EC2 A1 and Bare Metal platforms. Instance availability and installation documentation can be found in xref:../installing/installing-preparing.html#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms].
+
+The following features are supported for {product-title} on ARM:
+
+* OpenShift Cluster Monitoring
+* Currently, the following Operators are supported:
+** Node Tuning Operator
```

Comment:
Thoughts:

What if we rearranged this a little? 

Move 'Currently, the following Operators are supported:' and the list operators to the bottom to the bottom (after NFS storage on Bare Metal) and introduced a new list? 

So for example:

The following features are supported for {product-title} on ARM:
* OpenShift Cluster Monitoring
* Currently, the following Operators are supported:
* * RHEL 8 Application Streams
* OVNKube
* OpenShift Logging
* Elastic Book Store (EBS) for AWS
* AWS Certificate Manager (ACM)
* AWS .NET applications
* NFS storage on Bare metal

The following Operators are supported on ARM: 
** Node Tuning Operator
** Node Feature Discovery Operator
** Cluster Samples Operator
** Cluster Logging Operator
** Elasticsearch Operator
** Service Binding Operator

Alternatively you could put Operator support first, and then list features. This is just a thought and not a suggestion/requirement to move forward. Just playing with some ideas. I'm having a hard time coming to terms with how the second list (which is Operator specific) is introduced in the first list (which is Feature specific). Does that make sense?